packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+-   We updated the typings generator for icon properties
```

Comment:
We always explicitly mention the feature for which the changes are necessary (in this case, custom icon collections, right?) and the Mendix version in which that feature was added / made public (9.24).

The version number of the package should also be updated to reflect this.

Reply:
It might also be a good idea to mention that the existing `null` type has been replaced with `undefined` for compatibility with the `Icon` component in the public API. This is especially important if the actual output of the preview API has changed, as that is technically a breaking change (but one that I think is acceptable in this situation). It may cause existing TypeScript-based widgets to stop compiling.

It's sad that we cannot reuse the `WebIcon` type from the public API for this, due to the "compatibility" prop that should never have been there... ah well.

Reply:
See my other comment, I'll resolve this one.